src/Compilers/VisualBasic/Test/Semantic/Diagnostics/OperationAnalyzerTests.vb

```diff
@@ -2048,12 +2048,13 @@ End Module
                 Diagnostic(ERRID.ERR_BadIteratorReturn, "Sub").WithLocation(28, 21),
                 Diagnostic(ERRID.HDN_UnusedImportStatement, "Imports System.Collections.Generic").WithLocation(2, 1))
             comp.VerifyAnalyzerDiagnostics({New MemberReferenceAnalyzer}, Nothing, Nothing, False,
-                Diagnostic(MemberReferenceAnalyzer.HandlerAddedDescriptor.Id, "AddHandler, New Object").WithLocation(26, 24),
```

Comment:
Why get rid of the IDs?

Reply:
It seems the original test was hand crafted, and hence had the IDs. Failure text contains the string representation of the IDs, so should be easier to maintain.

---
In reply to: [193881051](https://github.com/dotnet/roslyn/pull/27579#discussion_r193881051) [](ancestors = 193881051)